Scripts/99 SL-ThemePrefs.lua

```diff
@@ -90,6 +90,11 @@ SL_CustomPrefs.Get = function()
 			Choices =  { THEME:GetString("ThemePrefs","Yes"), THEME:GetString("ThemePrefs", "No") },
 			Values	= { true, false }
 		},
+		KeyboardFeatures = {
+			Default = true,
```

Comment:
I think default should be false. Have operators manually enable it otherwise people might get locked up in a feature that maybe should not have been enabled.

Reply:
changed!